libsel4/include/interfaces/sel4.xml

```diff
@@ -132,6 +198,25 @@
                 description="Location of the thread's IPC buffer. Must be 512-byte aligned. The IPC buffer may not cross a page boundary."/>
             <param dir="in" name="bufferFrame" type="seL4_CPtr"
                 description="Capability to a page containing the thread's IPC buffer."/>
+            <error name="seL4_IllegalOperation">
+                <description>
+                    The <texttt text="_service,"/> <texttt text="bufferFrame,"/> <texttt text="cspace_root,"/> or <texttt text="vspace_root"/> is a CPtr to a capability of the wrong type.
+                    Or, <texttt text="vspace_root"/> is not assigned to an ASID pool.
+                    Or, <texttt text="cspace_root_data"/> is invalid.
+                    Or, <texttt text="buffer"/> is not aligned.
+                    Or, <texttt text="bufferFrame"/> is retyped from a device untyped <docref>(see <autoref label="sec:kernmemalloc"/>).</docref>
+                </description>
+            </error>
+            <error name="seL4_InvalidCapability">
+                <description>
+                    The <texttt text="_service"/> is a CPtr to a capability of the wrong type.
+                </description>
+            </error>
+            <error name="seL4_RevokeFirst">
+                <description>
+                    The <texttt text="bufferFrame,"/> <texttt text="cspace_root,"/> or <texttt text="vspace_root"/> is a CPtr to a capability of the wrong type.
+                </description>
+            </error>
```

Comment:
Maybe I missed an indirect path, but I don't think `TCBConfigure` can generate `seL4_RevokeFirst`

Reply:
found it..